test/integration/raw-queries.test.js

```diff
@@ -0,0 +1,266 @@
+'use strict';
+
+const chai = require('chai'),
```

Comment:
Can you please make that 1 const declaration per line?
Also please destructure `chai` so we don't have single use/proxy imports :)

Reply:
Done